tests/0.8.1/valid.json

```diff
@@ -0,0 +1,376 @@
+        "iss": "did:key:z6MkpWPEfw2yUXtmMz6SW4hye7XpjBQSN1EWje6t12mRDC7V",
+        "aud": "did:key:z6MkpkQPXHuxoqH7oukdohXbQ8uU8QE8KpAiuYpCEhwBvhff",
+        "exp": 4803890544,
+        "att": [],
+        "prf": []
+      }
+    }
+  },
+  {
+    "comment": "Payload `nbf` is valid",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtwV1BFZncyeVVYdG1NejZTVzRoeWU3WHBqQlFTTjFFV2plNnQxMm1SREM3ViIsImF1ZCI6ImRpZDprZXk6ejZNa3BrUVBYSHV4b3FIN291a2RvaFhiUTh1VThRRThLcEFpdVlwQ0Vod0J2aGZmIiwibmJmIjoxNjQ4MTMwNTQ0LCJleHAiOjQ4MDM4OTA1NDQsImF0dCI6W10sInByZiI6W119.7y78QBmdE6iEY7wzNR_z0VoJlRaSWPnP5Es_SzIKvS2SBcQaxB03bevov26DEdmQUPudQ0wWyEuU1GRa81uSCg",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkpWPEfw2yUXtmMz6SW4hye7XpjBQSN1EWje6t12mRDC7V",
+        "aud": "did:key:z6MkpkQPXHuxoqH7oukdohXbQ8uU8QE8KpAiuYpCEhwBvhff",
+        "nbf": 1648130544,
+        "exp": 4803890544,
+        "att": [],
+        "prf": []
+      }
+    }
+  },
+  {
+    "comment": "Payload `exp` is valid",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtwV1BFZncyeVVYdG1NejZTVzRoeWU3WHBqQlFTTjFFV2plNnQxMm1SREM3ViIsImF1ZCI6ImRpZDprZXk6ejZNa3BrUVBYSHV4b3FIN291a2RvaFhiUTh1VThRRThLcEFpdVlwQ0Vod0J2aGZmIiwiZXhwIjo0ODAzODkwNTQ0LCJhdHQiOltdLCJwcmYiOltdfQ.gTVI9hmLlv0JgiRYsuZ1xPxSn6T0fvyw_u8CbjaI1TW15RsqLQQv1j2aBeHk9TY7-hqNcuPCZFdgX2GgsgB6Aw",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkpWPEfw2yUXtmMz6SW4hye7XpjBQSN1EWje6t12mRDC7V",
+        "aud": "did:key:z6MkpkQPXHuxoqH7oukdohXbQ8uU8QE8KpAiuYpCEhwBvhff",
+        "exp": 4803890544,
+        "att": [],
+        "prf": []
+      }
+    }
+  },
+  {
+    "comment": "Payload `fct` is valid",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtwV1BFZncyeVVYdG1NejZTVzRoeWU3WHBqQlFTTjFFV2plNnQxMm1SREM3ViIsImF1ZCI6ImRpZDprZXk6ejZNa3BrUVBYSHV4b3FIN291a2RvaFhiUTh1VThRRThLcEFpdVlwQ0Vod0J2aGZmIiwiZXhwIjo0ODAzODkwNTQ0LCJmY3QiOlt7ImNoYWxsZW5nZSI6ImFiY2RlZiIsImZyb20iOiJleGFtcGxlLmNvbSJ9XSwiYXR0IjpbXSwicHJmIjpbXX0.enler8RlLavzZws_q92aYqbVY8XHdJesgtI7RcvO-c1zVLMzIOPl1zQv_R1tkUCSrMIy_BwvDf7tkYA44c6TCg",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkpWPEfw2yUXtmMz6SW4hye7XpjBQSN1EWje6t12mRDC7V",
+        "aud": "did:key:z6MkpkQPXHuxoqH7oukdohXbQ8uU8QE8KpAiuYpCEhwBvhff",
+        "exp": 4803890544,
+        "fct": [
+          {
+            "challenge": "abcdef",
+            "from": "example.com"
+          }
+        ],
+        "att": [],
+        "prf": []
+      }
+    }
+  },
+  {
+    "comment": "Delegated UCAN can have multiple valid proofs",
```

Comment:
What would you think about rephrasing this comment to "Delegated UCAN is valid with multiple valid proofs"? This would strengthen the idea that a UCAN can have multiple proofs _and_ is valid because of the capabilities granted by each of those proofs.